src/views/account/HeaderSection.tsx

```diff
@@ -19,21 +51,60 @@ export default function HeaderSection({ item, network }: any) {
                                 {NETWORK_LIST.find((el) => el.key === network)?.name}
                             </span>
                         </div>
-                        <div className="flex items-center flex-1 gap-2 break-words">
-                            <span className="text-sm leading-5 break-all text-dark-600"><Address text={item?.address}></Address></span>
-                            <CopyButton text={item?.address} />
-                            <button className="outline-none focus:outline-none ring-0 focus:ring-0">
-                                <Link
-                                    // underline="hover"
-                                    // color="text.primary"
-                                    href={`${NETWORK_SCANNER_MAP[network]}/address/${item?.address}`}
-                                    aria-current="page"
-                                    className="text-blue-200"
-                                    target="_blank"
-                                >
-                                    <img src={getExplorerLogo(network)} style={{height: '16px', width: '16px'}} alt="" />
-                                </Link>
-                            </button>
+                        <div className='w-full flex flex-col max-md:gap-[1rem] md:flex-row justify-between'>
+                            <div className="flex items-center flex-1 gap-2 break-words">
+                                <span className="text-sm leading-5 break-all text-dark-600">{item?.address}</span>
+                                <CopyButton text={item?.address} />
+                                <button className="outline-none focus:outline-none ring-0 focus:ring-0">
+                                    <Link
+                                        // underline="hover"
+                                        // color="text.primary"
+                                        href={`${NETWORK_SCANNER_MAP[network]}/address/${item?.address}`}
+                                        aria-current="page"
+                                        className="text-blue-200"
+                                        target="_blank"
+                                    >
+                                        <img src={getExplorerLogo(network)} style={{ height: '16px', width: '16px' }} alt="" />
+                                    </Link>
+                                </button>
+                            </div>
+                            <div className="flex flex-wrap items-center gap-1">
```

Comment:
See if this can be made into a component that can be shared across all the pages that have it. You can put the component in the global folder